src/plugins/template/src/compiled_model.cpp

```diff
@@ -77,10 +75,10 @@ void fill_output_info(const ov::Output<ov::Node>& output, InferenceEngine::DataP
 }  // namespace ov
 
 // ! [executable_network:ctor_cnnnetwork]
-TemplatePlugin::CompiledModel::CompiledModel(const std::shared_ptr<ov::Model>& model,
-                                             const std::shared_ptr<const ov::IPlugin>& plugin,
-                                             const std::shared_ptr<ov::threading::ITaskExecutor>& task_executor,
-                                             const Configuration& cfg)
+ov::template_plugin::CompiledModel::CompiledModel(const std::shared_ptr<ov::Model>& model,
```

Comment:
do we need a `plugin` word in namespace name?

Reply:
Unfortunatelly yes...(( `template` is a reserved name in C++ :)